src/js/common/schemaform/definitions/ssn.js

```diff
@@ -1,9 +1,13 @@
 import { validateSSN } from '../validation';
 import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+import SSNWidget from '../widgets/SSNWidget';
+import { TextWidget } from '../widgets';
 
 export const schema = commonDefinitions.ssn;
 
 export const uiSchema = {
+  'ui:widget': SSNWidget,
+  'ui:reviewWidget': TextWidget,
```

Comment:
Right widget, wrong file.

That formatting is something we've wanted to do before, so might as well add it.